src/airport.js

```diff
@@ -1 +1,49 @@
+class Airport {
+    hangar;
+    #maxCapacity;
+    weather;
 
+    constructor(plane = [], capacity = 3, weather = 'Clear') {
+        this.hangar = plane;
+        this.#maxCapacity = capacity;
+        this.weather = weather;
+    }
+
+    getWeather() {
+        return this.weather;
+    }
+
+    changeWeather(weather) {
+        weather === 'Stormy' ? this.weather = 'Stormy' : weather === 'Clear' ? this.weather = 'Clear' : console.log(`Please enter valid weather conditions. Valid conditions are 'Clear' or 'Stormy'.`);
+    }
+
+    getPlaneNum() {
+        return this.hangar.length;
+    }
+
+    getMaxCapacity() {
+        return this.#maxCapacity;
+    }
+
+    changeMaxCapacity(number) {
+        if (typeof number === 'number') {
+            return this.#maxCapacity = number
+        }
+    }
+    isAirportFull() {
+        return this.getMaxCapacity() === this.getPlaneNum()
+    }
+
+    landNewPlane(plane) {
+        this.hangar.includes(plane) ? console.log(`${plane.id} has already landed and is on standby for departure.`) : this.getWeather() === 'Stormy' ? console.log("Landing denied due to adverse weather conditions, please divert.") :
```

Comment:
EBI - this should be cleaner, if..else might be clearer code or at least each statement should be separate eg: 
question? option a 
: option b
: option c